packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -212,14 +227,10 @@ export class BlobManager {
      * about this for now because the data is a simple list of blob ids.
      */
     public getGCData(fullGC: boolean = false): IGarbageCollectionData {
```

Comment:
I'm really thinking we need to have a GC interface or maybe even an abstract class that has getGCData as the core of its functionality. We have this method implemented implicitly everywhere, but not explicitly. I think this should be explicit.

Reply:
I don't see how having an abstract class will help as layers that do want to participate in GC will be very different. Having an interface is part of the long term plan for GC - https://github.com/microsoft/FluidFramework/issues/8578.
It's not a priority for GC phase 1 which is focused on implementing mark and sweep phase for data stores and attachment blobs. In GC phase 2, we will have this interface and GC will be extended to support non-runtime controlled objects as well.